sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/presentation/screens/molecules/PasswordInputViewScreen.kt

```diff
@@ -0,0 +1,114 @@
+                placeholderText = stringResource(id = R.string.password_input_placeholder_placeholder),
+            )
+        }
+
+        ExamplesSlot {
+            HmrcCardView {
+                PasswordInputView(
+                    value = example1Value,
+                    modifier = Modifier.padding(
+                        horizontal = HmrcTheme.dimensions.hmrcSpacing16,
+                        vertical = HmrcTheme.dimensions.hmrcSpacing24,
+                    ),
+                    onInputValueChange = {
+                        viewModel.isEmptyValidation(it, errorText, 0)
+                        example1Value = it
+                    },
+                    errorText = viewModel.textInputErrorEmptyValidation.collectAsStateWithLifecycle().value,
+                    labelText = stringResource(id = R.string.password_input_example_1_label),
+                    hintText = stringResource(id = R.string.password_input_example_1_hint),
+                    maxChars = 4,
+                    numericOnly = true,
+                    errorContentDescription = stringResource(R.string.password_input_example_2_error_content_description),
+                    requiredSequencesSpacing = true
+                )
+
+                PasswordInputView(
+                    value = example2Value,
+                    modifier = Modifier.padding(
+                        horizontal = HmrcTheme.dimensions.hmrcSpacing16,
+                        vertical = HmrcTheme.dimensions.hmrcSpacing24,
+                    ),
+                    onInputValueChange = {
+                        viewModel.isEmptyValidation(it, errorText, 1)
+                        example2Value = it
+                    },
+                    errorText = viewModel.textInputErrorEmptyValidation1.collectAsStateWithLifecycle().value,
+                    labelText = stringResource(id = R.string.password_input_example_2_label),
+                    hintText = stringResource(id = R.string.password_input_example_2_hint),
+                    requiredSequencesSpacing = true
+                )
+
+                PasswordInputView(
+                    modifier = Modifier.padding(
+                        horizontal = HmrcTheme.dimensions.hmrcSpacing16,
+                        vertical = HmrcTheme.dimensions.hmrcSpacing24,
+                    ),
+                    value = example3Value,
+                    onInputValueChange = {
+                        viewModel.isEmptyValidation(it, errorText, 2)
+                        example3Value = it
+                    },
+                    errorText = viewModel.textInputErrorEmptyValidation2.collectAsStateWithLifecycle().value,
+                    labelText = stringResource(id = R.string.password_input_example_3_label),
+                    hintText = stringResource(id = R.string.password_input_example_3_hint),
+                )
+            }
```

Comment:
Could we add an example that showcases all the accessibility stuff so it can be tested, like adding different content descriptions and button text?

Reply:
Put it all on ex1 now